packages/gitgraph-core/src/branch.ts

```diff
@@ -74,10 +74,11 @@ export class Branch {
       options.parents.unshift(this.parentCommit.hash);
     }
 
+    const { tag, ...commitOptions } = options;
```

Comment:
Just realized that we have forgot the syntax `gitgraph.commit({tag: "1.0.0"})` :wink: